site/content/en/latest/tasks/observability/gateway-api-metrics.md

```diff
@@ -2,57 +2,134 @@
 title: "Gateway API Metrics"
 ---
 
-Resource metrics for Gateway API objects are available using the [Gateway API State Metrics][gasm] project.
-The project also provides example dashboard for visualising the metrics using Grafana, and example alerts using Prometheus & Alertmanager.
+Resource metrics for **Kubernetes Gateway API** objects are available through the [Gateway API State Metrics][gasm] project. The project also includes an example dashboard for visualizing the metrics with Grafana, along with sample alerts using Prometheus and Alertmanager.
 
 ## Prerequisites
 
-{{< boilerplate prerequisites >}}
+{{< boilerplate o11y_prerequisites >}}
 
-Run the following commands to install the metrics stack, with the Gateway API State Metrics configuration, on your kubernetes cluster:
+### Enable kube-state-metrics
+
+The `kube-state-metrics` service is required to collect metrics from the Kubernetes API server. Use the following command to enable it:
 
 ```shell
-kubectl apply --server-side -f https://raw.githubusercontent.com/Kuadrant/gateway-api-state-metrics/main/config/examples/kube-prometheus/bundle_crd.yaml
-kubectl apply -f https://raw.githubusercontent.com/Kuadrant/gateway-api-state-metrics/main/config/examples/kube-prometheus/bundle.yaml
+helm upgrade eg-addons oci://docker.io/envoyproxy/gateway-addons-helm \
+--version {{< helm-version >}} \
+--reuse-values \
+--set prometheus.kube-state-metrics.enabled=true \
```

Comment:
I don't think this will be sufficient. 
you also need enable custom resource monitoring in kube-state-metrics
See: https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-state-metrics/values.yaml#L452

Reply:
Okay, I see that now. Thx.